packages/mobile/src/transactions/TransferFeedItem.tsx

```diff
@@ -285,4 +285,5 @@ const styles = StyleSheet.create({
   },
 })
 
+// @ts-ignore
```

Comment:
what the error that is being ignored ?

Reply:
The error is a bit cryptic. I think it could be fixed by upgrading react-i18next type definition. But not entirely sure.

```
$ tsc --noEmit
src/transactions/TransferFeedItem.tsx:288:55 - error TS2345: Argument of type 'typeof TransferFeedItem' is not assignable to parameter of type 'ComponentType<TransferStandby & WithNamespaces & { status?: TransactionStatus | undefined; invitees: Invitees; addressToE164Number: AddressToE164NumberType; recipientCache: NumberToRecipient; commentKey: Buffer | null; }>'.
  Type 'typeof TransferFeedItem' is not assignable to type 'ComponentClass<TransferStandby & WithNamespaces & { status?: TransactionStatus | undefined; invitees: Invitees; addressToE164Number: AddressToE164NumberType; recipientCache: NumberToRecipient; commentKey: Buffer | null; }, any>'.
    Type 'TransferFeedItem' is not assignable to type 'Component<TransferStandby & WithNamespaces & { status?: TransactionStatus | undefined; invitees: Invitees; addressToE164Number: AddressToE164NumberType; recipientCache: NumberToRecipient; commentKey: Buffer | null; }, any, any>'.
      Types of property 'props' are incompatible.
        Type '(Readonly<TransferStandby & WithNamespaces & { status?: TransactionStatus | undefined; invitees: Invitees; addressToE164Number: AddressToE164NumberType; recipientCache: NumberToRecipient; commentKey: Buffer | null; }> & Readonly<...>) | (Readonly<...> & Readonly<...>)' is not assignable to type 'Readonly<TransferStandby & WithNamespaces & { status?: TransactionStatus | undefined; invitees: Invitees; addressToE164Number: AddressToE164NumberType; recipientCache: NumberToRecipient; commentKey: Buffer | null; }> & Readonly<...>'.
          Type 'Readonly<HomeTransferFragment & WithNamespaces & { status?: TransactionStatus | undefined; invitees: Invitees; addressToE164Number: AddressToE164NumberType; recipientCache: NumberToRecipient; commentKey: Buffer | null; }> & Readonly<...>' is not assignable to type 'Readonly<TransferStandby & WithNamespaces & { status?: TransactionStatus | undefined; invitees: Invitees; addressToE164Number: AddressToE164NumberType; recipientCache: NumberToRecipient; commentKey: Buffer | null; }> & Readonly<...>'.
            Property 'id' is missing in type 'Readonly<HomeTransferFragment & WithNamespaces & { status?: TransactionStatus | undefined; invitees: Invitees; addressToE164Number: AddressToE164NumberType; recipientCache: NumberToRecipient; commentKey: Buffer | null; }> & Readonly<...>' but required in type 'Readonly<TransferStandby & WithNamespaces & { status?: TransactionStatus | undefined; invitees: Invitees; addressToE164Number: AddressToE164NumberType; recipientCache: NumberToRecipient; commentKey: Buffer | null; }>'.

288 export default withNamespaces(Namespaces.walletFlow5)(TransferFeedItem)
                                                          ~~~~~~~~~~~~~~~~

  src/transactions/reducer.ts:17:3
    17   id: string
         ~~
    'id' is declared here.


Found 1 error.
```